tests/jvm/src/test/resources/example/MacroAnnotations_2.13.scala

```diff
@@ -1,5 +1,5 @@
 package example
 
 @MacroAnnotation/*=>example.MacroAnnotations#*/
-class MacroAnnotations/*<=example.MacroAnnotations#*/
+class MacroAnnotations
```

Comment:
and here the marking is removed for some reason

Reply:
And I think this one should also get an issue